src/test/groovy/graphql/schema/validation/TypesImplementInterfacesTest.groovy

```diff
@@ -20,13 +18,6 @@ import static graphql.schema.GraphQLUnionType.newUnionType
 
 class TypesImplementInterfacesTest extends Specification {
 
-    TypeResolver typeResolver = new TypeResolver() {
```

Comment:
These are older tests, `typeResolver` not required directly on interface/union type